manifests/server.pp

```diff
@@ -1,34 +1,34 @@
 # See README.md for details.
 class openldap::server (
-  $package                                          = $openldap::params::server_package,
-  $confdir                                          = $openldap::params::server_confdir,
-  $conffile                                         = $openldap::params::server_conffile,
-  $service                                          = $openldap::params::server_service,
+  String[1] $package,
+  String[1] $confdir,
+  String[1] $conffile,
+  String[1] $service,
+  String[1] $owner,
+  String[1] $group,
+  Boolean $escape_ldapi_ifs,
+  Optional[Boolean] $enable_chown,
```

Comment:
```suggestion
  Optional[Boolean] $enable_chown = undef,
```

Reply:
Tanks! I added an extra commit to handle these.